src/plugins/zoom-image.js

```diff
@@ -10,13 +10,17 @@ function install(hook) {
       ),
     );
 
+    Docsify.dom.style(
+      `.medium-zoom-image--opened,.medium-zoom-overlay{z-index:999}`,
+    );
+
```

Comment:
This changes gonna overlap the whole screen as the zoom out canvas, does it intent to?

Reply:
Be consistent with v4